package.json

```diff
@@ -8,7 +8,7 @@
     "cypress-setup": "./cypress-setup.sh",
     "clean-e2e-projects": "git checkout e2e-projects/ && git clean -f e2e-projects/",
     "prepare": "husky install",
-    "postinstall": "lerna bootstrap --no-private && npm install --prefix e2e-projects/next",
```

Comment:
don't remove the `--no-private` flag, or we'll end up with alpha versions of deprecated packages